pkgs/top-level/all-packages.nix

```diff
@@ -26654,7 +26654,9 @@ with pkgs;
   grafana = callPackage ../servers/monitoring/grafana { };
   grafanaPlugins = callPackages ../servers/monitoring/grafana/plugins { };
 
-  grafana-agent = callPackage ../servers/monitoring/grafana-agent { };
+  grafana-agent = callPackage ../servers/monitoring/grafana-agent {
+    buildGoModule = buildGo121Module;
```

Comment:
Please describe in the commit message and as a comment here why we're explicitly using `buildGo121Module` instead of `buildGoModule`.

Reply:
Whoops, sorry!

Did amend the commit message just now.
Feel free to click the "Resolve conversation" button, if you feel like this change is good enough :)

Reply:
The commit message says `<=`, while it should be `>=`, no?

Reply:
whoops (again).
fixed now :)